### PR TITLE
SUNBURST Azure Enhancement 

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added manual action for - 
1. Hunting suspicious activity with Azure.
2. Configuration review for Azure
3. More mitigation actions

Created a new pack for Microsoft Policy And Compliance (Audit Log)

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed.

**Review by bakatzir — COMMENTED** [review]
@altmannyarden FYI, the deps are calculated in the upload to the bucket process. so if you want to add extra manually that is ok, but it will be calc and updated anyhow(PBs that use commands from integrations and scripts, mappers and classifiers with incident fields, etc)

**Comment by altmannyarden** [comment]
@bakatzir when I wanted to test the pack - I didn't see this pack as a dependency - that's why I added manually. 
image

**Comment by bakatzir** [comment]
@altmannyarden Yes, because it is calculated in the flow upload to the bucket as I wrote.
if you want to check what will be added. you can run the `find-dependencies` command. no need to add manually (And if it is in the command deps, it will have no additional affect anyhow).

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_Activity_Behavior_Hunting_playbook.yml [hunk 1]
```diff
@@ -0,0 +1,463 @@
+id: SolarStorm Activity Behavior Hunting playbook
```
Comment (David-BMS) [inline Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-SolarStorm_Activity_Behavior_Hunting_playbook.yml:1]: @kirbles19 @ShirleyDenkberg Does it make sense for the playbook name to contain the word playbook?

**Review by David-BMS — APPROVED** [review]
This review included just the following playbooks
* Azure Hunting playbook
* SolarStorm Activity Behavior Hunting playbook
* Azure Configuration Analysis

Other files are not relevant at @altmannyarden request.

**Pull request merged**